src/EventService.Models.Db/Models/DbCategory.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+using LT.DigitalOffice.EventService.Models.Db.Enums;
+
+namespace LT.DigitalOffice.EventService.Models.Db.Models
+{
+  public record DbCategory
+  {
```

Comment:
It is better to add new variable 'TableName'  to all dbModels. Like here https://github.com/LT-Students/DigitalOffice-DepartmentService/blob/develop/src/DepartmentService.Models.Db/DbCategory.cs